graph/graph_test.go

```diff
@@ -301,6 +301,6 @@ func tempGraph(t *testing.T) (*Graph, graphdriver.Driver) {
 }
 
 func nukeGraph(graph *Graph) {
```

Comment:
Actually for `Root`, that case in push should just be calling `s.graph.RawJSON`, and not that it takes an ID instead of an image the signature matches :smile:. I will go ahead and unexport it.